### ISSUE TITLE
Elasticsearch crashes after Couchdb River Initiates

### ISSUE THREAD

**Issue by biswajitdas** [elasticsearch/elasticsearch-river-couchdb#82]
Hi,
I am using Elasticsaerch 1.4.1
```{
status: 200,
name: "Tutinax the Mountain-Mover",
cluster_name: "elasticsearch",
version: {
number: "1.4.1",
build_hash: "89d3241d670db65f994242c8e8383b169779e2d4",
build_timestamp: "2014-11-26T15:49:29Z",
build_snapshot: false,
lucene_version: "4.10.2"
},
tagline: "You Know, for Search"
}```
Whenever I am trying to create a river with a filter
```
{"type":"couchdb","couchdb":{"host":"192.168.10.31","port":"5984","db":"database_name","filter":"view\/view_name"},"index":{"index":"index_name","type":"type_name","bulk_size":"100","bulk_timeout":"10ms"}}
```
Response
```
{"_index":"_river","_type":"type_name","_id":"_meta","_version":1,"created":true}
```
the [elasticsearch/Head](https://github.com/mobz/elasticsearch-head) stops loading. I found, when I am creating this the ```curl -XGET 192.168.10.31:9200/state``` comes nothing, where the head fails. If I delete the _river it again resume loading. Is this strange?
But the same is working fine with 1.2.1 version of Elasticsaearch.

**Comment by emptyflask** [comment]
This appears to be the same issue at #78, which is fixed in the repo but as far as I can tell, no released version. You should be able to get it working if you build 2.5.0-snapshot.

**Comment by dadoonet** [comment]
2.4.2 will be released really soon. It has this fix #81 which hopefully fix the issue.
